great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -217,10 +218,26 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
-        if not (0.0 <= false_positive_rate <= 1.0):
+
+        if false_positive_rate < 0 or false_positive_rate >= 1:
```

Comment:
@NathanFarmer equal to 1 should be OK (the minus EPSILON from the positive direction should handle the equals).  Thanks.

Reply:
👍🏻  ✅